tokio-postgres/src/error/mod.rs

```diff
@@ -309,7 +309,14 @@ impl DbError {
 
 impl fmt::Display for DbError {
     fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(fmt, "{}: {}", self.severity, self.message)
+        write!(fmt, "{}: {}", self.severity, self.message)?;
+        if let Some(detail) = &self.detail {
+            write!(fmt, "DETAIL: {}", detail)?;
```

Comment:
Shouldn't there be some whitespace before DETAIL and HINT?

Reply:
Good catch, I added a newline before each.